config/locales/plurals.rb

```diff
@@ -1 +1,9 @@
-GovukI18n.plurals
+GovukI18n.plurals.merge(
```

Comment:
Would it be worth adding these to https://github.com/alphagov/govuk_app_config/blob/main/lib/govuk_app_config/govuk_i18n.rb ?

Reply:
We've previously been using [this unicode document](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html) as a definitive definition of the pluralisation rules.

I'd suggest we should update `govuk_app_config` only (so we don't lose the benefits of previous work that moved everything there, rather than having different fragmented rules across various apps) and if we think the translations in other apps are wrong (based on the unicode definitions), then we should go back to the translations firm to query why they didn't include these plurals.

Reply:
That makes sense — starting work on government-frontend I've noticed a similar set of issues, so it looks like it's not limited to Whitehall and I'm going to be extracting this change.

However I think what's happened in most cases is actually that we have _extra_ translations (singular and plural for a language where the rule says only plural), and that sometimes these are different in ways that I'm not entirely comfortable deleting (e.g., more than just capitalisation).

Reply:
Based on that document, all of the languages I've added here ought to be changed except for Vietnamese — which as noted above is mostly just duplicated except for a few (about 4) singulars which differ from the plural.